app/Http/Transformers/ActionStatTransformer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\ActionStat;
+use League\Fractal\TransformerAbstract;
+
+class ActionStatTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\ActionStat $actionStat
+     * @return array
+     */
+    public function transform(ActionStat $actionStat)
+    {
+        $result = $actionStat;
+        unset($result->deleted_at);
```

Comment:
This should [return an array](https://fractal.thephpleague.com/transformers/) with the format we want for this API endpoint. This ensures that if we add/remove/rename properties down the line, we can keep the same HTTP contract (and we don't need to worry about an expanding list of properties to `unset` here either!